lang/cs/Org.Apache.REEF.Network/Group/Driver/Impl/GroupCommDriver.cs

```diff
@@ -170,10 +198,10 @@ public IConfiguration GetServiceConfiguration()
                     GenericType<ICodec<GroupCommunicationMessage>>.Class,
                     GenericType<GroupCommunicationMessageCodec>.Class)
                 .BindNamedParameter<NamingConfigurationOptions.NameServerAddress, string>(
-                    GenericType<NamingConfigurationOptions.NameServerAddress>.Class, 
+                    GenericType<NamingConfigurationOptions.NameServerAddress>.Class,
```

Comment:
This entire class doesn't need to be changed anymore. But when revert it back, there is an extra space removed so still showing a change.